app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -63,6 +64,10 @@
     inputmode = "numeric"
     pattern = "[0-9]*"
   end
+
+  if type == "email"
+    spellcheck = "false"
+  end
```

Comment:
I'm just wondering if we should also include `type == "number"`? (I see that this type does get modified to `text` like so):
```
  if type == "number"
    type = "text"
    inputmode = "numeric"
    pattern = "[0-9]*"
  end
```
... but looking at the example it feels like we should probably set `spellcheck = "false"` too. Like for the 'account number example https://components.publishing.service.gov.uk/component-guide/input/numeric_input/preview

Reply:
I like it, let me update.